src/router/routes.ts

```diff
@@ -1,33 +1,95 @@
+// import { RouteRecordRaw } from 'vue-router';
+
+// const routes: RouteRecordRaw[] = [
+//   {
+//     path: '/',
+//     component: () => import('layouts/MainLayout.vue'),
+//     children: [
+//       { path: '', component: () => import('pages/Home.vue'), name: 'Home' },
+
+//       {
+//         path: '/about',
+//         component: () => import('pages/About.vue'),
+//         name: 'About',
+//       },
+
+//       {
+//         path: '/wines',
+//         component: () => import('pages/Wines.vue'),
+//         name: 'Wines',
+//       },
+
+//       {
+//         path: '/food',
+//         component: () => import('src/pages/Food/Food.vue'),
+//         name: 'Food',
+//       },
+//       {
+//         path: '/food/starters',
```

Comment:
did you meen to use the path /food/starters or do you want just /starters?

Reply:
i meen /food/starters :)